src/multiformats/hashes/index.ts

```diff
@@ -0,0 +1,16 @@
+import { from } from 'multiformats/hashes/hasher'
+import * as digest from 'multiformats/hashes/digest'
+import { BeeSon } from '../../beeson'
+
+export const encode = async (hashBytes: Uint8Array): Promise<Uint8Array> => {
```

Comment:
why do you call the parameter `hashBytes`? it is just the BeeSon's byte representation that has nothing to do with hashes.

Reply:
this is what they name it in swarm-cids.....